File: src/klassen/Zelle.java
package klassen;

public class Zelle extends Thread {

    private SpielDesLebens spiel;
    private byte[] position;
    private boolean isAlive;
    public Zelle(byte[] position, SpielDesLebens spiel) {

        this.position = position;
        isAlive = true;
        this.spiel=spiel;

    }

    public byte[] getPosition() {
        return position;
    }

    public void setPosition(byte[] position) {
        this.position = position;
    }

    public boolean isAlivee() {
        return isAlive;
    }

    public void setAlive(boolean alive) {
        isAlive = alive;
    }

    @Override
    public void run() {

        //System.out.println(Thread.currentThread()+" gestartet");
        try {
            Thread.sleep(100);
        } catch (InterruptedException e) {
            e.printStackTrace();
        }
        while(!isInterrupted()){
                isAlive=spiel.getCellAliveStatus(this);

        }
    }

}

File: Zelle.py
import threading
from time import sleep
from turtle import setposition

class Zelle(threading.Thread):
    def __init__(self, spiel, position):
        self.position = position
        self.spiel = spiel
        self.is_Alive = True

    def getPosition(self):
        return self.position

    def setPosition(self, position):
        self.position = position

    def isAlive(self):
        return self.isALive

    def setAlive(self, alive):
        self.isALive = alive

    def run(self):
        try:
            sleep(100)
        except InterruptedError:
            print("Fehler")
        #while(self.is_Alive):
            #self.spiel <- here call the Funktion in the SpielDesLebens Class
        
    def stop_Tread(self):
        self.is_Alive = False
    #Siehe https://openbook.rheinwerk-verlag.de/python/32_005.html
    #Und https://docs.python.org/3/library/threading.html#condition-objects
